system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/TransferPullEndToEndTest.java

```diff
@@ -167,47 +171,59 @@ void httpPull_dataTransfer_withEdrCache() {
             providerDataSource.verify(request("/source").withMethod("GET"));
         }
 
-        @Test
-        void suspendAndResumeByConsumer_httpPull_dataTransfer_withEdrCache() {
+        private static Stream<Participant> participants() {
+            return Stream.of(CONSUMER, PROVIDER);
+        }
+
+        @ParameterizedTest(name = "Suspend and resume httpPull dataTransfer with Edr cache by {0}")
+        @MethodSource("participants")
```

Comment:
parameterized tests needs to be used wisely, especially in the E2E context. 
In this case I don't think it is a good use for them, if the problem is the test code duplication I suggest extracting a method with the common logic and keep the two tests separated, because they represent two different scenarios that is better to represented separately.